mona_sdk/client.py

```diff
@@ -323,8 +323,13 @@ def _get_user_id(self):
         """
         :return: The customer's user id (tenant id).
         """
+        current_token = get_current_token_by_api_key(self.api_key)
+        if not current_token:
+            return None
```

Comment:
is it safe to return None as user_id? isn't it used somewhere important that epxects it to exist? It's possible to test that it raises an exception too in case this behavior is the right one btw